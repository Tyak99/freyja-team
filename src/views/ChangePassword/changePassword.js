import React from 'react';
// eslint-disable-next-line import/no-named-as-default
import ChangePassword from '../../components/ChangePassword/ChangePassword';

const ResetPassword = () => (
  <ChangePassword />
);


export default ResetPassword;
