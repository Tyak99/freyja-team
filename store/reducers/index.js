import { combineReducers } from 'redux';
import { createBrowserHistory } from 'history';
import profile from './profile';
import authReducer from './authReducer';
import modulesReducer from './modulesReducer';
import communityM from './community';

export const history = createBrowserHistory();

const appReducer = combineReducers({
  auth: authReducer,
  profile,
  modules: modulesReducer,
  community: communityM,
});

export default (state, action) => appReducer(state, action);
