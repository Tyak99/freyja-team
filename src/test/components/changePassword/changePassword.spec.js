import React from 'react';
import { shallow } from 'enzyme';
import { ChangePassword } from '../../../components/ChangePassword/ChangePassword';
import store from '../../../../store';

let ResetPassword;

describe('Reset password component', () => {
  it('should render correctly the reset password component', () => {
    ResetPassword = shallow(<ChangePassword store={store} />);
    expect(ResetPassword).toMatchSnapshot();
  });
});
