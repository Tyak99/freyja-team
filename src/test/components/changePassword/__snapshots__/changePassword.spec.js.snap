// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset password component should render correctly the reset password component 1`] = `ShallowWrapper {}`;
